pcdsdevices/areadetector/detectors.py

```diff
@@ -15,6 +17,10 @@
 from ophyd.device import Component as Cpt
 from ophyd.signal import EpicsSignal, EpicsSignalRO
 
+from pcdsdevices.variety import set_metadata
+
+from pcdsdaq.ext_scripts import hutch_name
```

Comment:
`pcdsdevices` does not have a `pcdsdaq` dependency, nor should it. This is why the travis is breaking.
Should we move these utilties to `pcdsutils`?

Reply:
Sounds good to me.

Reply:
Ok I will transplant `ext_scripts` to `pcdsutils`